test/compile/legend.test.ts

```diff
@@ -36,6 +72,14 @@ describe('Legend', function() {
   });
 
   describe('properties.symbols', function() {
-    // FIXME test
+    it('should initialize if filled', function() {
+      const symbol = legend.properties.symbols({field: 'a'}, {}, parseUnitModel({
+          mark: "point",
```

Comment:
add `export` keyword in front of `namespace` on line
https://github.com/vega/vega-lite/blob/zl/legend-properties/src/compile/legend.ts#L121